include/dpp/role.h

```diff
@@ -74,45 +74,71 @@ class DPP_EXPORT role : public managed, public json_interface<role> {
 	 * @brief Role name
 	 * Between 1 and 100 characters.
 	 */
-	std::string name;
+	std::string name{};
 	/**
 	 * @brief Guild ID
 	 */
-	snowflake guild_id;
+	snowflake guild_id{0};
```

Comment:
no need for this change to the class, makes the PR bigger.
We usually set defaults in the constructor, not via {} in the header.